test/cpp/tensorexpr/test_schedule.cpp

```diff
@@ -432,16 +432,16 @@ void testScheduleFuserThreeArg() {
   Buffer c(Var("C", kHandle), kFloat32, {Expr(kTotalSize)});
   Buffer d(Var("D", kHandle), kFloat32, {Expr(kTotalSize)});
 
-  Tensor e = Compute(
+  Tensor* e = Compute(
       "e", {{kTotalSize, "i"}}, [&](const Var& i) { return a(i) + b(i); });
-  Tensor f = Compute(
-      "f", {{kTotalSize, "i"}}, [&](const Var& i) { return e(i) + c(i); });
-  Tensor g = Compute(
-      "g", {{kTotalSize, "i"}}, [&](const Var& i) { return f(i) + d(i); });
+  Tensor* f = Compute(
+      "f", {{kTotalSize, "i"}}, [&](const Var& i) { return (*e)(i) + c(i); });
```

Comment:
Looks good in general. Minor nit: if everything is pointer, "e->call(i)" looks slightly better than (*e)(i).

Thanks for the cleanup!